2018/Furmanets/Championship/TournamentsWpfForms/WindowAddPlayers.xaml.cs

```diff
@@ -39,20 +43,38 @@ private void PlayerNameBox_OnMouseEnter(object sender, MouseEventArgs e)
         private void StartDouble_Click(object sender, RoutedEventArgs e)
         {
             var tournament = new DoubleEliminationTournament(_players);
+            _fileManager.WriteToFile(tournament);
             OpenWindowTournament(tournament);
         }
 
         private void StarSingle_Click(object sender, RoutedEventArgs e)
         {
             var tournament = new SingleElimitationTournament(_players);
+            _fileManager.WriteToFile(tournament);
             OpenWindowTournament(tournament);
         }
 
         private void OpenWindowTournament(Tournament tournament)
         {
-            var window = new TournamentPlayWindow(tournament);
+            var window = new TournamentPlayWindow(tournament, _fileManager);
             window.Show();
             Close();
         }
+
+        private void PlayerNameBox_OnKeyDown(object sender, KeyEventArgs e)
+        {
+            switch (e.Key)
+            {
+                case Key.Enter:
+                    AddPlayerButtom(sender, e);
+                    break;
+                case Key.Down:
+                    StartSingleTournamentButton.Focus();
+                    break;
+                case Key.Up:
+                    StartDoubleTournamentButton.Focus();
+                    break;
+            }
+        }
     }
 }
```

Comment:
I consider these names (and 1 message) to have some mistakes connected with spelling and wrong order of words: nextCursorPositionLeft, nextDistanceBeewinPalyers,
 nextCursorPositionTop, maxLengthName, indexForDrowLine,
 positionCursorLeft, positionCursorTop, PlaceForDrawind,
CanvasForPrintTournamentGrid, AddPlayerButtom,
"All matches played."